website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx

```diff
@@ -155,7 +157,41 @@ If you've set `enableAutoEncrypt: true`, this is also supported.
 With the above settings added to your existing config, follow the [Upgrading](/docs/k8s/operations/upgrading)
 guide to upgrade your cluster and then come back to the [Federation Secret](#federation-secret) section.
 
--> **NOTE:** You must be using consul-helm 0.21.0+. To update, run `helm repo update`.
```

Comment:
nit: did you mean to remove "To update, run `helm repo update`"?

Reply:
Yeah because that support came out in May last year so I figured it wasn't necessary.